### PR TITLE
[JS] Topic: Events in JavaScript

### LINKED ISSUE #761: "Events" in javascript.
[Path Name-]  (https://github.com/girlscript/winter-of-contributing/tree/Javascript/Web_Development/Javascript/Topics)

One of the important concept of javascript. The change in the state of an object is known as an Event. This process of reacting over the events is called Event Handling. Thus, js handles the HTML events via Event Handlers. These are some terms related to events.

I will contribute through code, document and image.
I am a GWOC'21 contributor from **Batch-1 of javascript** Please assign this issue to me. I assure you that this issue's title will be the same as my PR's title as well as I will be Contributing under the Guidelines and Code of Conduct.

### LINKED ISSUE #2319: JS: Events in JavaScript
# Description

- What is an Event ?
- Mouse events
- Keyboard events
- onclick Event Type
- onsubmit Event Type

# Directory
```Topics/15.Events```

### Domain

JavaScript

### Type of Contribution

Documentation

### Code of Conduct

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by sushovanb02**
<hr>

## Description 📜

Created a document on Events in JavaScript
Fixes #761 ~#2319~

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by sushovanb02** [comment]
@paramsiddharth @s4-2 @Creatoon @dakshsethi please review this PR

**Review by paramsiddharth — APPROVED** [review]

File: JavaScript/Topics/15.Events/README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+# HTML 5 Standard Events
+
+| Attribute          | Value  | Description                                                                                                  |
+|--------------------|--------|--------------------------------------------------------------------------------------------------------------|
+| Offline            | script | Triggers when the document goes offline                                                                      |
+| Onabort            | script | Triggers on an abort event                                                                                   |
+| onafterprint       | script | Triggers after the document is printed                                                                       |
+| onbeforeonload     | script | Triggers before the document loads                                                                           |
+| onbeforeprint      | script | Triggers before the document is printed                                                                      |
+| onblur             | script | Triggers when the window loses focus                                                                         |
```
Comment (dakshsethi) [inline JavaScript/Topics/15.Events/README.md:83]: Link all of these attributes to their W3Schools demo page.
For Example
`[onblur](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onblur)`

**Review by dakshsethi — CHANGES_REQUESTED** [review]

**Review by dakshsethi — CHANGES_REQUESTED** [review]
Add all events with their links.

**Comment by Creatoon** [comment]
@dakshsethi Write a generalized example, and show, who is event listener, who is event handler and what's the event, also briefly describe event listener and handler

File: JavaScript/Topics/15.Events/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+// In JavaScript:
+object.onsubmit = function(){myScript};
+// In JavaScript, using the addEventListener() method:
+object.addEventListener("submit", myScript);
+```
+
+Example:
+
+```js
+// Execute a JavaScript when a form is submitted:
```
Comment (Creatoon) [inline JavaScript/Topics/15.Events/README.md:66]:
```suggestion
// Execute `myFunction()` when the form is submitted:
```

**Review by Creatoon — CHANGES_REQUESTED** [review]

**Comment by sushovanb02** [comment]
> @sushovanb02
> Write a generalized example, and show, who is event listener, who is event handler and what's the event, also briefly describe event listener and handler

ok

File: JavaScript/Topics/15.Events/README.md (hunk 2 above)
Comment (sushovanb02) [reply JavaScript/Topics/15.Events/README.md]: @Creatoon check now

**Review by Creatoon — APPROVED** [review]

**Comment by sushovanb02** [comment]
> Add all events with their links.

@dakshsethi done, now you may check

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

**Comment by sushovanb02** [comment]
@paramsiddharth all reviews done i guess you can merge this now

**Pull request merged**